test/client/fixtures/transport-test.js

```diff
@@ -100,7 +101,7 @@ if (!browserUtils.isWebKit && !browserUtils.isFirefox) {
             });
     });
 }
-else {
+else if (!isSaucelabs) {
```

Comment:
Why we need to get off these tests for saucelabs? Do all the tests are unstable?